tests/Feature/PostTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Comment;
+use App\Models\Like;
+use App\Models\User;
+use App\Models\Post;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class PostTest extends TestCase
+{
+    use DatabaseMigrations;
+    /**
+     * A basic test example.
+     */
+    public function test_user_can_create_post(): void
+    {
+        $user = User::factory()->create();
+        Post::create([
+            'user_id'      => $user->id,
+            'post_content' => 'hello'
+        ]);
+
+        $this->assertDatabaseHas('posts', 
+        [
+            'user_id'      => $user->id,
+            'post_content' => 'hello'
+        ]);
+    }
+
+    /**
+     * A basic test example.
+     */
+    public function test_user_can_create_comment(): void
+    {
+        $user = User::factory()->create();
+        $post = Post::factory()->create();
+        Comment::create(
+            [
+                'user_id'      => $user->id,
+                'comment_text' => 'comment here on hello',
+                'post_id'      => $post->id
+            ]);
+            $this->assertDatabaseHas('comments', 
+            [
+                'user_id'      => $user->id,
+                'comment_text' => 'comment here on hello',
+                'post_id'      => $post->id
+            ]);
```

Comment:
These should be aligned like this - so the closing ")" aligns with the opening "("

```suggestion
        Comment::create(
            [
                'user_id'      => $user->id,
                'comment_text' => 'comment here on hello',
                'post_id'      => $post->id
            ]
        );
        $this->assertDatabaseHas('comments', 
            [
                'user_id'      => $user->id,
                'comment_text' => 'comment here on hello',
                'post_id'      => $post->id
            ]
        );
```